### ISSUE TITLE
Remove unused fonts for Android

### ISSUE THREAD

**Issue by housseindjirdeh** [gitpoint/git-point#485]
Similar to #475, we'll need to do the same in Android (RNE has react-native-vector-icons as a dependency and linking the library includes _all_ which we aren't using)

**Comment by dyesseyumba** [comment]
Hello! I want to work on this.

**Comment by dyesseyumba** [comment]
When I try to commit, I've got this error.
```
cat: $GIT_PARAMS: No such file or directory
D:\Repositories\OS\git-point\node_modules\@commitlint\cli\cli.js:118
                throw err;
                ^

TypeError: Expected a raw commit
    at parser (D:\Repositories\OS\git-point\node_modules\conventional-commits-parser\lib\parser.js:17:11)
    at sync (D:\Repositories\OS\git-point\node_modules\conventional-commits-parser\index.js:96:10)
    at _callee$ (D:\Repositories\OS\git-point\node_modules\@commitlint\core\lib\library\parse.js:40:16)
    at tryCatch (D:\Repositories\OS\git-point\node_modules\regenerator-runtime\runtime.js:65:40)
    at Generator.invoke [as _invoke] (D:\Repositories\OS\git-point\node_modules\regenerator-runtime\runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as next] (D:\Repositories\OS\git-point\node_modules\regenerator-runtime\runtime.js:117:21)
    at step (D:\Repositories\OS\git-point\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at D:\Repositories\OS\git-point\node_modules\babel-runtime\helpers\asyncToGenerator.js:28:13
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

husky > commit-msg hook failed (add --no-verify to bypass)
C:\Users\Dyesse YUMBA\AppData\Local\Yarn\config\global\node_modules\commitizen\dist\cli\strategies\git-cz.js:102
        throw error;
        ^

Error: git exited with error code 1
    at ChildProcess.<anonymous> (C:\Users\Dyesse YUMBA\AppData\Local\Yarn\config\global\node_modules\commitizen\dist\git\commit.js:50:26)
    at emitTwo (events.js:125:13)
    at ChildProcess.emit (events.js:213:7)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:200:12)

D:\Repositories\OS\git-point>

```
Can somebody help me?

I'm using Windows 10, node 8.7 and yarn 1.2.1

**Comment by machour** [comment]
@dyesseyumba I'm not on windows, but you can add `--no-verify` to your commit to bypass commitlint

**Comment by housseindjirdeh** [comment]
Thank you so much @dyesseyumba 🙌 And WELCOME to the family.

Yeah we have a git commit hook that has errored for a few individuals time to time 😞  In the meantime if your commit follows the convention manually, you can use `--no-verify` as @machour suggested until we can figure out why you're seeing that error.

**Comment by machour** [comment]
Closed in #512 by @dyesseyumba 
❤️
